src/main/assembly/dist/cfg/application.properties

```diff
@@ -6,21 +6,27 @@
 daemon.http.port=20190
 
 #
-# Directories for managing the deposits. The lifecycle of a deposit is:
+# Directories for managing the deposits.
 #
-# 1.   Created in the drafts directory.
-# 2.   On submit: copied to stage and then that copy is moved to the submit-to directory.
-# 3a)  If archived successfully the bag in the draft-copy is deleted
-# 3b)  If the deposit is rejected, the depositor can reopen the draft, modify it and resubmit it.
+# - stage-zips:       is used for uploads of zipped directories or complete deposits.
+# - drafts:           contains the draft deposits; single files are uploaded here directly,
+#                     unzipped directories and deposits are moved here from stage-zips.
+# - stage-for-submit: the completed draft is copied here when the user submits it.
```

Comment:
perhaps add a couple of words on why a '_completed draft_' must first be moved to `stage-for-submit`, before it is copied to the `submit-to` directory. That wasn't clear to me on first reading.

Reply:
side note, I see this documentation is duplicated in `src/test/resources`. Please keep them in sync!
If necessary, add a test for that (although I can imagine you've already done so 😉)

Reply:
DebugConfigSpec checks for the same files in both config directories and for the same keys in both application.properties files. The values are different. Don't know how to check for comments.

Perhaps reduce comment in debug-config to a reference to `dist/config`?

Reply:
How about reading the file for the 'comment test' as lines, filtering the ones that start with `#` and comparing the resulting output.

```scala
val srcFile = srcConfig.lines.filter(_ startsWith "# ").toList
val testFile = testConfig.lines.filter(_ startsWith "# ").toList

srcFile should contain theSameElementsInOrderAs testFile
```

Reply:
If you really want that comment check: create an issue for all 16 modules that have a `DebugConfigSpec`. You may come up with more checks.

Reply:
> side note, I see this documentation is duplicated in `src/test/resources`. Please keep them in sync!
> If necessary, add a test for that (although I can imagine you've already done so 😉)

I think there is little value in keeping the comments in `src/test/resources`. I'd say: just leave them out of those. They are just for development testing and developers should know where to find them in `src/main/assembly`.

Reply:
I'm happy with dropping the comment duplication.

The archteype has the following header in both `dist/config` as in `debug-config`.
```
#
# EASY Module configuration file
#
# Please, modify the configuration below to match your environment. Also, delete and/or modify
# this description.
```
* Shouldn't this comment header be dropped in `debug-config` of the archetype too?
* Should the archetype replace `Module` in `dist/config` with the module name?
* Should we synchronise the chosen convention in all our scala modules?